gtk-4.yaml

```
@@ -70,12 +70,12 @@ environment:
       - orc-dev
       - pango-dev
       - py3-docutils
-      - py3-gi-docgen
+      - py3-gi-docgen~2024.1-r2
       - py3-packaging
       - sassc
       - shaderc
       - shared-mime-info
-      - smartypants
+      - smartypants~2.0.1-r3
```

```suggestion
      - smartypants
```
Epoch bump should enough, it should fetch the latest version with latest epoch. pinning this could result in older version in future.